### ISSUE TITLE
 CSS highlight pseudo ::search-text

### ISSUE THREAD

**Issue by jihyerish** [WebKit/standards-positions#421]
### WebKittens

@smfr @annevk 

### Title of the proposal

CSS Pseudo-Elements Module Level 4 § Highlight Pseudo-elements § Selecting Highlighted Content

### URL to the spec

https://drafts.csswg.org/css-pseudo-4/#selectordef-search-text

### URL to the spec's repository

https://github.com/w3c/csswg-drafts/tree/main/css-pseudo-4

### Explainer URL

https://github.com/Igalia/explainers/blob/main/css/find-in-page/README.md

### TAG Design Review URL

N/A

### Mozilla standards-positions issue URL

https://github.com/mozilla/standards-positions/issues/1103

### Description

This feature allows for CSS styling of the search results shown using a browser’s find-in-page feature.

The Chromium intent to prototype is [here](https://groups.google.com/a/chromium.org/g/blink-dev/c/5FS_4mVQBLc/m/vm-WWXUCAQAJ)

**Comment by jihyerish** [comment]
This feature allows modifying a new highlight style for the matched "find-in-page" strings, which are rendered differently across browsers.
@fantasai, could you give an opinion about this?

**Comment by hober** [comment]
Safari's rendering of find-in-page search results is not representable in CSS, so such a pseudo seems somewhat unlikely to be useful.

**Comment by schenney-chromium** [comment]
Right. In the CSS meeting  where the feature was discussed it was agreed that Safari would not implement due to a very different find-in-page display that already addresses page contrast and so on. I guess we're just asking if you're against other browsers implementing it.
